rules/source-control-tools/rule.md

```diff
@@ -0,0 +1,143 @@
+---
+type: rule
+title: Do you know some Source Control tools ?
+uri: source-control-tools
+authors:
+  - title: Gilles Pothieu
+    url: https://ssw.com.au/people/gilles-pothieu/
+created: 2024-01-30T13:44:07.470Z
+guid: 87b1677f-72c8-4b6f-9976-d3cfab075272
+---
+Version control is an essential tool for software developers. It enables tracking changes made to a project's source code, facilitates collaboration among developers, and ensures the project's consistency.
+
+<!--endintro-->
+
+Version control is an essential tool for software developers. It enables tracking changes made to a project's source code, facilitates collaboration among developers, and ensures the project's consistency.
+
+### Advantages of Version Control Software
+
+1. **Project Consistency Assurance:** it keeps a record of all changes made to a project's source code. This allows reverting to a previous version of the code in case of problems.
```

Comment:
Hi @PothieuG 

For ALL lists:

- [ ] 1. Remove the repeated "it" from every list item...

- [ ] 2. Can you change all colons to be dashes?

- [ ] 3. Make sure the first letter after the dash is in uppercase.

E.g. from ": it keeps" to " - Keeps"

- [ ] 4. Remove all full stops at the end. As per https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists